Dockerfile

```diff
@@ -31,10 +31,19 @@ RUN find /public \
   -name 'index.html' \
   -delete
 
+COPY src/robots.txt /public/robots.txt
+
 FROM gsoci.azurecr.io/giantswarm/nginx:1.23-alpine
 
 COPY nginx.conf /etc/nginx/nginx.conf
 
+# Development mode
+# RUN sed -i \
+#     -e 's|listen       80;|listen 8080;|' \
+#     -e 's|listen  [::]:80;||' \
+#     /etc/nginx/conf.d/default.conf
```

Comment:
I cannot make it work to test it (robots.txt without removing that line)

Reply:
What exact error do you observe? Server not starting?

Reply:
Something like `docker-entrypoint.sh not found when executed inside a container` and then I cannot access the server from my local